src/staging/src/components/pageContent/linkCard.js

```diff
@@ -34,13 +34,12 @@ export default function LinkCard({ url, title, subTitle, imageUrl, imageAltText,
       cardClasses += ""
       break;
   }
-
   return (
     <div className={wrapClasses}>
     <Link
```

Comment:
Getting warnings in the console about using the `Link` component for external URLs.   May want to create a link wrapper component to switch between `Link` and regular anchor elements

Reply:
Interesting. I'll touch base with Cole on that as he introduced that in a prev branch

Reply:
Creating a switch component is probably the correct solution. In the short term we could switch to just using a tags, it's a bit slower with internal links but still works.

Reply:
will switch to <a> for this ticket, and add a tech debt ticket to look into `Link`